packages/renderer/src/lib/kube/pods/pod-utils.ts

```diff
@@ -50,6 +50,7 @@ export class PodUtils {
 
     return {
       name: pod.metadata?.name ?? '',
+      kind: 'pods',
```

Comment:
Here also (`Pod`), and in the future (after #10916), we may double-check that it is the same as `pod.kind`